services/tasks/project/api/tasks.py

```diff
@@ -127,32 +139,15 @@ def populate_db():
     ]
 
     response_object = dict()
-
-    # make get request to DX_IP
-    # we will have a list of issues for each repo and we will add that to the json response object
-    # which is a dictionary
-    # issues = {status: success, repo1: [list of issues 1], repo2: [list of issues 2], ...}
-    # response_object = {'status' : 'success'}
     client = Client(host="http://{}".format(os.environ.get('DX_IP')))
```

Comment:
`client` no longer needed here. Or should be passed to `get_items`. Don't know how heavy it is.

Reply:
Great catch, removed!